NitroxServer/Communication/Packets/Processors/Abstract/TransmitIfCanSeePacketProcessor.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections.Generic;
+using NitroxModel.DataStructures;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.Packets;
+using NitroxServer.GameLogic;
+using NitroxServer.GameLogic.Entities;
+
+namespace NitroxServer.Communication.Packets.Processors.Abstract;
+
+public abstract class TransmitIfCanSeePacketProcessor<T> : AuthenticatedPacketProcessor<T> where T : Packet
+{
+    private readonly PlayerManager playerManager;
+    private readonly EntityRegistry entityRegistry;
+
+    public TransmitIfCanSeePacketProcessor(PlayerManager playerManager, EntityRegistry entityRegistry)
+    {
+        this.playerManager = playerManager;
+        this.entityRegistry = entityRegistry;
+    }
+
+    /// <summary>
+    /// Transmits the provided <paramref name="packet"/> to all other players (excluding <paramref name="senderPlayer"/>)
+    /// who can see (<see cref="Player.CanSee"/>) entities corresponding to the provided <paramref name="entityIds"/> only if all those entities are registered.
+    /// </summary>
+    public void TransmitIfCanSeeEntities(Packet packet, Player senderPlayer, List<NitroxId> entityIds)
+    {
+        List<Entity> entities = [];
+        foreach (NitroxId entityId in entityIds)
+        {
+            if (entityRegistry.TryGetEntityById(entityId, out Entity entity))
+            {
+                entities.Add(entity);
+            }
+            else
+            {
+                return;
```

Comment:
This would probably be a useless spam giving no other info that some creatures are present on a client but not on the server. As we're aware of this issue, it's not required that we log anything concerning it